presto-main/src/main/java/io/prestosql/execution/TaskManager.java

```diff
@@ -77,6 +77,16 @@
      */
     ListenableFuture<TaskStatus> getTaskStatus(TaskId taskId, TaskState currentState);
 
+    /**
+     * Changes the {@link io.prestosql.memory.MemoryPool} which query must be assigned.
+     * <p>
+     * Presto has two memory pools:
+     * - General pool: It is used by default.
+     * - Reserved pool: It is used to run the heaviest query when
```

Comment:
```suggestion
     * - Reserved pool: It is used to run the heaviest queries when
```

Reply:
The reserved pool just run one query per time, i think just keep in singular